common/djangoapps/student/models.py

```diff
@@ -505,6 +505,8 @@ class Meta(object):
     allow_certificate = models.BooleanField(default=1)
     bio = models.CharField(blank=True, null=True, max_length=3000, db_index=False)
     profile_image_uploaded_at = models.DateTimeField(null=True, blank=True)
+    phone_regex = RegexValidator(regex=r'^\+?1?\d*$', message="Phone number can only contain numbers.")
```

Comment:
Is it worth writing a unit test for this regex?

Reply:
Are we only allowing US based phone numbers?

Reply:
The plan is to allow for international numbers on the model, and do any additional filtering elsewhere.

Reply:
@crice100 unit test added!